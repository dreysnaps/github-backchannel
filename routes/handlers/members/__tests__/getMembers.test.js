jest.mock("../../../lib/octokit");

const { getMembers } = require("../getMembers");

describe("Check Valid Org", () => {
  it("Check members of org which exist (github)", async () => {
    expect(await getMembers({ params: { org: "github" } })).toMatchSnapshot();
  });
});
