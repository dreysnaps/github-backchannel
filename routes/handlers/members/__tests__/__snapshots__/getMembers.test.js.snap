// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check Valid Org Check members of org which exist (github) 1`] = `
Object {
  "members": Array [
    Object {
      "avatar_url": "https://avatars0.githubusercontent.com/u/50083?v=4",
      "followers": 395,
      "following": 1,
      "login": "abinoda",
    },
    Object {
      "avatar_url": "https://avatars3.githubusercontent.com/u/11968290?v=4",
      "followers": 13,
      "following": 0,
      "login": "abhishekhegde",
    },
  ],
}
`;
